src/plugins/ircColors/index.ts

```diff
@@ -66,8 +66,8 @@ export default definePlugin({
         {
             find: '="SYSTEM_TAG"',
             replacement: {
-                match: /\i.gradientClassName]\),style:/,
-                replace: "$&{color:$self.calculateNameColorForMessageContext(arguments[0])},_style:"
+                match: /(style:)(.{0,500}textDecorationColor)/,
```

Comment:
this is very bad

Reply:
```suggestion
                match: /(style:)(\(\(\)=>\{)/,
```

this is what i did, pick your poison